agent/cache/cache_test.go

```diff
@@ -609,7 +609,7 @@ func TestCacheGet_periodicRefreshBadRPCZeroIndexErrorBackoff(t *testing.T) {
 // immediately on the initial request if there is no data written to that table
 // yet.
 func TestCacheGet_noIndexSetsOne(t *testing.T) {
-	t.Parallel()
+	// t.Parallel()
```

Comment:
This was either a temporary flake for me when testing `make cov` or I'll fix it elsewhere and rebase.